test-models/src/main/java/com/mx/testing/accessors/AccountBaseAccessor.java

```diff
@@ -69,4 +69,38 @@ public AccessorResponse<ModelList<com.mx.testing.models.v20240101.Account>> list
   public AccessorResponse<Account> get(String id) {
     throw new AccessorMethodNotImplementedException();
   }
+
+  /**
+   * Get account by id
+   * @param id
+   * @return
+   */
+  @GatewayAPI
+  @API(description = "Get user's account by account", version = "20240101")
+  public AccessorResponse<com.mx.testing.models.v20240101.Account> get20240101(String id) {
+    throw new AccessorMethodNotImplementedException();
+  }
+
+  /**
+   * Create account
+   * @param account
+   * @return
+   */
+  @GatewayAPI
+  @API(description = "Create account")
+  public AccessorResponse<Account> create(Account account) {
+    throw new AccessorMethodNotImplementedException();
+  }
+
+  /**
+   * Create account
+   * @param account
+   * @return
+   */
+  @GatewayAPI
+  @API(description = "Create account", version = "20240101")
+  @RemoteOperation("create20240101")
```

Comment:
What is this annotation for?

Reply:
This is how we set this function apart from the other one (with the same name) when calling it remotely. The RemoteService generator requires it. This won't matter since we are no longer using NATs.